vuln/npm/519.json

```
@@ -0,0 +1,24 @@
+{
+  "id": 519,
+  "title": "Denial of Service",
+  "overview": "Crash Node.js process from handlebars using a small and simple source",
+  "created_at": "2019-10-31",
+  "updated_at": "2020-04-27",
+  "publish_date": "1970-01-01",
```

Looks like the issue is public on H1 now
```suggestion
  "publish_date": "2020-04-27",
```